### PR TITLE
Spp 9629 empty field processing

### PR DESCRIPTION
**Pull request by JasonBellONS**
## Synopsis

Adding in additional default checks to better handle empty values within the method

## Checklist

- [x] Documentation created/updated
- [x] Tests created/updated

## Description

Add a more detailed description of the pr including any necessary background
information.

### CODE REVIEW

File: sml_small/utils/pandas_example.py [hunk 1]
```diff
@@ -84,9 +85,8 @@ def load_csv(filepath):
     upper_limit_column="threshold_upper",
     lower_limit_column="threshold_lower",
     predictive_column="predictive_val",
-    auxiliary_column="aux_val",
```
Comment (gibbardsteve) [inline sml_small/utils/pandas_example.py:87]: Why aren't we passing auxiliary through?

**Review by gibbardsteve — COMMENTED** [review]
There are also comments from Ellie about renaming some values and omitting some data from the return. Maybe we can include them as part of this ticket (see email thread)

File: sml_small/utils/pandas_example.py (hunk 1 above)
Comment (JasonBellONS) [reply sml_small/utils/pandas_example.py:87]: my bad I used this branch for the demo with Ellie last week and must have forgotten to undo the changes to this file when showing thousand pounds working with "true optional" values, amended

**Comment by JasonBellONS** [comment]
@gibbardsteve i've amended the aux value in the py-example and made those changes requested by Ellie

**Review by gibbardsteve — APPROVED** [review]

**Pull request merged**